.gitignore

```diff
@@ -13,3 +13,6 @@ CmakeSettings.json
 CombinedSources*.cpp
 .DS_Store
 *~
+/cmake-build-debug
+/cmake-build-release
+/.idea
```

Comment:
"cmake-build-debug" and "cmake-build-release" shouldn't be in .gitignore since they are specific to your build. The .idea can remain in .gitignore as this might benefit more people.

Reply:
I use clion ide and it copies files in this folders. GitHub is olweys chalking the folders if it could add then to commit and so on. Therefore in gitignore I ignored builds folder for clion for better stability and working. It shouldn't do any bad think to anyone through because I doubt tgui would use them.

Reply:
I didn't realize that the ide created those debug and release folders too, I though you created those and the ide only created the .idea folder.
Since those folder names seem to be the default setting for clion, you can keep these folders in .gitignore too.